notebook/static/terminal/js/main.js

```diff
@@ -45,32 +38,17 @@ requirejs([
     }
     ws_url = ws_url + base_url + ws_path;
     
-    var header = $("#header")[0];
-
-    function calculate_size() {
-        var height = $(window).height() - header.offsetHeight;
-        var width = $('#terminado-container').width();
-        var rows = Math.min(1000, Math.max(20, Math.floor(height/termRowHeight())-1));
-        var cols = Math.min(1000, Math.max(40, Math.floor(width/termColWidth())-1));
-        console.log("resize to :", rows , 'rows by ', cols, 'columns');
-        return {rows: rows, cols: cols};
-    }
-    
     page.show_header();
     
-    var size = calculate_size();
-    var terminal = terminado.make_terminal($("#terminado-container")[0], size, ws_url);
+    var terminal = terminado.make_terminal($("#terminado-container")[0], ws_url);
     
     page.show_site();
     
     utils.load_extensions_from_config(config);
     utils.load_extensions_from_config(common_config);
     
-    window.onresize = function() { 
-      var geom = calculate_size();
-      terminal.term.resize(geom.cols, geom.rows);
-      terminal.socket.send(JSON.stringify(["set_size", geom.rows, geom.cols,
-                                    $(window).height(), $(window).width()]));
```

Comment:
We still need to send the new size to the server so that it can trigger a resize in the running process. Try running a full-screen program like `vim` or `aptitude`, and resize the terminal while it's running.